setup.cfg

```diff
@@ -29,10 +34,12 @@ platforms =
   POSIX
 
 [options]
+use_scm_version = True
```

Comment:
This is legacy and doesn't really mean anything nowadays with the modern packaging toolchain.
```suggestion
```

Reply:
indeed, this is already implied by `[tool.setuptools_scm]` in `pyproject.toml`